examples/purchase-tester-java/src/main/java/com/revenuecat/purchasetester/java/UpsellActivity.java

```diff
@@ -107,9 +107,9 @@ public void onClick(View v) {
 
     private void makePurchase(Package packageToPurchase, final Button button) {
         showLoading(button, true);
-        Purchases.getSharedInstance().purchasePackage(this, packageToPurchase, new MakePurchaseListener() {
+        Purchases.getSharedInstance().purchasePackage(this, packageToPurchase, new PurchaseCallback() {
             @Override
-            public void onCompleted(@NonNull Purchase purchase, @NonNull CustomerInfo customerInfo) {
+            public void onCompleted(@NonNull PaymentTransaction purchase, @NonNull CustomerInfo customerInfo) {
```

Comment:
i'm just remembering now that @aboedo and i talked about renaming this to StoreTransaction...andy, can you confirm what our decision was?

Reply:
We ended up going with `StoreTransaction` (during a pairing session with @beylmk), and entirely failed to communicate it to the rest of the team 🤦 

Reasoning was: 
- consistency with naming of `StoreProduct`
- We realized that we already have a concept of `Transaction`, so we felt like we might need to make the difference clearer, so we figured we could use `Store` to differentiate stuff that's from the Stores (i.e.: App Store, Play Store, Amazon Store) vs stuff that's from us. 

@vegaro @taquitos @NachoSoto thoughts?

Reply:
Renamed!